UnityProject/Assets/Scripts/HealthV2/Living/Damage/Trauma/TraumaTypes/TraumaBurning.cs

```diff
@@ -80,7 +80,7 @@ public override void ProgressDeadlyEffect()
 		/// </summary>
 		private void AshBodyPart()
 		{
-
+			if (bodyPart.BodyPartType == BodyPartType.Head) return;
```

Comment:
why?
Heads should be able to burn and melt.

Reply:
and what is this fixing? It isn't mentioned anywhere

Reply:
so, it causes your entire body to basically get removed by flames